parquet-avro/src/main/java/parquet/avro/AvroParquetReader.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Read Avro records from a Parquet file.
  */
-public class AvroParquetReader<T extends IndexedRecord> extends ParquetReader<T> {
+public class AvroParquetReader<T> extends ParquetReader<T> {
```

Comment:
Is this a binary compatible change? If not, then we may need a separate class. Similarly, for AvroParquetWriter, AvroReadSupport and AvroWriteSupport.